### PR TITLE
Create moduleExample.lua

### PR DESCRIPTION
**Pull request by supertobi**
lua_gui_example

### CODE REVIEW

File: examples/moduleExample.lua [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+dt.configuration.check_version(...,{3,0,0})
+
+-- add a new lib
+local check_button = dt.new_widget("check_button"){label = "MyCheck_button", value = true}
+local combobox = dt.new_widget("combobox"){label = "MyCombobox", value = 2, "8", "16", "32"}
+local entry = dt.new_widget("entry"){text = "test", placeholder = "placeholder"}
+local file_chooser_button = dt.new_widget("file_chooser_button"){title = "MyFile_chooser_button"}
+local label = dt.new_widget("label")
+label.label = "MyLabel"
```
Comment (houz) [inline examples/moduleExample.lua]: Can't that be put into `{}`?

File: examples/moduleExample.lua [hunk 2]
```diff
@@ -0,0 +1,85 @@
+--[[
+    This file is part of darktable,
+    copyright (c) 2016 Tobias Ellinghaus
```
Comment (houz) [inline examples/moduleExample.lua]: Sure that's my code and not yours?

Comment (supertobi) [inline examples/moduleExample.lua]: Fixed

File: examples/moduleExample.lua (hunk 1 above)
Comment (supertobi) [inline examples/moduleExample.lua]: {} would work too, but I added it to show the alternative syntax

**Pull request merged**